lib/bindings/MQTTBinding.js

```diff
@@ -255,7 +255,8 @@ function start(callback) {
         clean: typeof mqttConfig.clean === 'boolean' ? mqttConfig.clean : true,
         clientId: mqttConfig.clientId ? mqttConfig.clientId + rndSuffix : 'iotajson' + rndSuffix,
         keepalive: mqttConfig.keepalive ? parseInt(mqttConfig.keepalive) : 60,
-        connectTimeout: 60 * 60 * 1000
+        connectTimeout: 60 * 60 * 1000,
+        subscribeBatchSize: 8
```

Comment:
According with doc:

subscribeBatchSize : optional number Maximum number of topics per SUBSCRIBE packet. When the number of topics to subscribe exceeds this value, the client will automatically split them into multiple SUBSCRIBE packets of this size.

but what is the default value used by subscribeBatchSize if none is provided ? Not sure if this new conf should be added into mqttConfig.